### PR TITLE
Lesson1 - Add directory with files

### PR DESCRIPTION
**Pull request by lyppay**

### CODE REVIEW

File: Lesson1_business-card/styles.css [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+    gap: 0px;
+    opacity: 0px;  
+    display: flex;  
+    flex-direction: column;
+}
+
+.name-data{
+    width: 383px;
+    height: 190px;
+    /* padding: 44px 206px 35px 19px; okomentowane bo chce o nie zapytac*/
```
Comment (mwanago) [inline Lesson1_business-card/styles.css]: Don't use Polish, even in the comments

File: Lesson1_business-card/styles.css [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+    border-bottom: solid 1px white;
+    display: flex;
+    flex-direction: column;
+}
+
+.contact-data{
+    width: 363px;
+    height: 132px;
+    top: 190px;
+    /* padding: 29px 0px 29px 20px; okomentowane bo chce o nie zapytac*/
```
Comment (mwanago) [inline Lesson1_business-card/styles.css]: Don't use Polish, even in the comments

File: Lesson1_business-card/styles.css [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+
+.text{
+    font-family: Roboto;
+    text-align: left;   
+}
+
+/*podkreslenie tylko do konca tekstu
+ .street-data, .email-data, .phone-data{
+    border-bottom: solid 1px white;
+} */
```
Comment (mwanago) [inline Lesson1_business-card/styles.css]: Please don't leave code that's commented like that

File: Lesson1_business-card/styles.css [hunk 4]
```diff
@@ -0,0 +1,127 @@
+.page{
```
Comment (mwanago) [inline Lesson1_business-card/styles.css]: There should be spaces before `{`

**Review by Solnick — CHANGES_REQUESTED** [review]
Please adjust your code to the comments made by @mwanago earlier

File: Lesson1_business-card/styles.css [hunk 5]
```diff
@@ -0,0 +1,107 @@
...
+    line-height: 29.3px;
+    width: 147px;
+    color: #FFFFFF;
+    font-weight: 400;
+}
+
+.surname{
+    font-size: 30px;
+    font-weight: 500;
+    line-height: 35.16px;
```
Comment (Solnick) [inline Lesson1_business-card/styles.css:77]: don't use pixels with decimal points.

File: Lesson1_business-card/styles.css [hunk 6]
```diff
@@ -0,0 +1,107 @@
...
+    color: #FFFFFF;
+}
+
+.work{
+    width: 157px;
+    height: 29px;
+    line-height: 29.3px;
+    padding: 19px 206px 0px 19px;
+    color: #6487A1; 
+    font-family: Roboto;
```
Comment (Solnick) [inline Lesson1_business-card/styles.css:87]: set the font for the whole body element. It should work and spare you setting the font-family in every text-element

File: Lesson1_business-card/styles.css [hunk 7]
```diff
@@ -0,0 +1,107 @@
...
+    text-align: left;   
+}
+
+.street, .email, .phone{
+    display: flex;
+    gap: 10px;
+}
+
+
+.street_txt, .email_txt, .phone_txt{
```
Comment (Solnick) [inline Lesson1_business-card/styles.css]: use kebab-case for class names

File: Lesson1_business-card/styles.css (hunk 4 above)
Comment (lyppay) [reply Lesson1_business-card/styles.css]: line added

File: Lesson1_business-card/styles.css (hunk 3 above)
Comment (lyppay) [reply Lesson1_business-card/styles.css]: comment removed

File: Lesson1_business-card/styles.css (hunk 2 above)
Comment (lyppay) [reply Lesson1_business-card/styles.css]: comment removed

File: Lesson1_business-card/styles.css (hunk 1 above)
Comment (lyppay) [reply Lesson1_business-card/styles.css]: comment removed

**Comment by lyppay** [comment]
> Please adjust your code to the comments made by @mwanago earlier

DONE

File: Lesson1_business-card/styles.css (hunk 4 above)
Comment (lyppay) [reply Lesson1_business-card/styles.css]: fixed

File: Lesson1_business-card/styles.css (hunk 1 above)
Comment (lyppay) [reply Lesson1_business-card/styles.css]: (I am resolving this, because conversation died)

File: Lesson1_business-card/styles.css (hunk 2 above)
Comment (lyppay) [reply Lesson1_business-card/styles.css]: (I am resolving this, because conversation died)

File: Lesson1_business-card/styles.css (hunk 3 above)
Comment (lyppay) [reply Lesson1_business-card/styles.css]: (I am resolving this, because conversation died)

File: Lesson1_business-card/styles.css (hunk 4 above)
Comment (lyppay) [reply Lesson1_business-card/styles.css]: (I am resolving this, because conversation died)

**Comment by lyppay** [comment]
I have done merge to master because there was no action from long time

File: Lesson1_business-card/styles.css (hunk 5 above)
Comment (lyppay) [reply Lesson1_business-card/styles.css:77]: done

File: Lesson1_business-card/styles.css (hunk 6 above)
Comment (lyppay) [reply Lesson1_business-card/styles.css:87]: done

File: Lesson1_business-card/styles.css (hunk 7 above)
Comment (lyppay) [reply Lesson1_business-card/styles.css]: done
(kebab-case xDDDD)

**Pull request merged**